### PR TITLE
Submit effort

### PR DESCRIPTION
**Pull request by gordoncd**
modified submit.php so that there's radio buttons to select type of file uploaded.  Also connected to our operational db in google drive using file/folder id's.

### CODE REVIEW

**Comment by gordoncd** [comment]
didn't get chance to test this, so please do so before merging

**Comment by thecodingaviator** [comment]
@gordoncd just tested it out. It looks like it is uploading an empty file to my drive. Let me look at the code real quick and see if I can provide more info on what is doing what

File: submit.php [hunk 1]
```diff
@@ -153,36 +167,60 @@ function handleSignoutClick() {
...
       const fileID = response.result.id;
-      console.log(fileID);
-
-      const token = gapi.client.getToken().access_token;
-
-      await fetch(`https://www.googleapis.com/upload/drive/v3/files/${fileID}?uploadType=media`, {
-        method: 'PATCH',
-        headers: new Headers({ 'Authorization': `Bearer ${token}` }),
-        body: file,
-      });
```
Comment (thecodingaviator) [inline submit.php:176]: Uploading to drive happens in two steps and it confused the heck out of me, so I'm gonna provide my best to explain how I understand it. I believe you first make a file and then upload the body of the file. I believe that the code above uploads the body, so we definitely need to keep it

**Review by thecodingaviator — CHANGES_REQUESTED** [review]

**Comment by thecodingaviator** [comment]
Closes #3

**Pull request closed without merging**